src/patterns/addresses/index.md

```diff
@@ -65,7 +65,11 @@ In production, you’ll need to do this to meet [WCAG 2.1 Level AA](https://www.
 
 Error messages should be styled like this:
 
-{{ example({group: "patterns", item: "addresses", example: "error", html: true, nunjucks: true, open: false, size: "s"}) }}
+{{ example({group: "patterns", item: "addresses", example: "error-pattern-whole", html: true, nunjucks: true, open: false, size: "xl"}) }}
```

Comment:
My only thought is whether `error-pattern-whole` is descriptive enough of an ID for the new code example. Not a huge deal either way though.

Other options might be `error-messages`, `errors-required-address-inputs`, or `errors-address`

Reply:
I'll go with error-messages